### DISCUSSION TITLE
동영상 촬영 예제에서 질문있어요.

### DISCUSSION THREAD

**Discussion by zlmrzlo** [naverz/zepeto-studio-kor#1257] [5. Q&A]
출시된 월드 중에서 불가리맵에서 동영상을 촬영하고 촬영한 동영상을 다시 재생해주던데

그렇게 하기 위해서 제공된 예제가 있나요? 이미 동영상 촬영 예제는 확인한 상태입니다.

아니면 최소한 어느 부분을 만져야 촬영한 동영상을 다시 재생할 수 있는지 너무 궁금합니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 가이드에 있는 예제 코드 중 아래 부분이 this.videoPlayerObject를 통해 녹화한 비디오를 재생하는 코드입니다. 

```typescript
*CheckRecording() {
        while (WorldVideoRecorder.IsRecording()) {
            yield null;
        }

        let videoPlayer = WorldVideoRecorder.AddVideoPlayerComponent(this.videoPlayerObject, this.targetTexture);

        if (videoPlayer == null)
            return;

        videoPlayer.Play();
    } 


```

다만 this.videoPlayerObject가 Canvas내에 있어야 눈에 보이므로 아래와 같이 세팅해 보세요.
우선 샘플 코드와 다르게 public으로 선언

```typescript
    /* VideoPlayer Object */
    public videoPlayerObject: GameObject;

    Awake() {
        //this.videoPlayerObject = new GameObject();
    }

```

캔버스에서 videoPlayerObject 위치 세팅

<img width="1077" alt="Screen Shot 2022-09-05 at 8 02 44 PM" src="https://user-images.githubusercontent.com/100118650/188434883-865a3804-1aba-4867-b882-19a8a3a22648.png">

그 후 녹화를 해보시면 미리보기를 확인하실 수 있습니다.

감사합니다.

**Reply by zlmrzlo** [reply]
감사합니다!
